docs/HowTo/Transactions/Make-Transactions.md

```diff
@@ -33,7 +33,7 @@ Transaction object for private transactions:
 
 | Key             | Type                | Required/Optional                  | Value                                                                                                                         |
 |-----------------|--:-:----------------|----------------------------------- |-------------------------------------------------------------------------------------------------------------------------------|
-| **from**        | Data, 20&nbsp;bytes | Required                           | Address of the sender. Must be the address of the keystore account.                                                                                                        |
+| **from**        | Data, 20&nbsp;bytes | Required                           | Address of the sender. Must be the address of the keystore account. If using [multiple key files](../../Concepts/Multiple-Key-Files.md), the keystore file name must contain the sender's address.                           |
```

Comment:
I think it isn't necessary to link the field definitions to the multi-file rules. I wouldn't mention it here.

Reply:
I agree with Lucas here - this is a related to the overall linking the signing key to the creating of key files.  ie, if we mention multifile here then we'd also be mentioning Azure and Hashicorp

Reply:
Done.